BILLthebuilder/automart
# #166398357 Setup failing tests for user registration

* It creates a test for making a reservation to a meetup
#### Description of Task to be completed?
* The `POST/api/v1/auth/signup` endpoint is tested here
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ch-user-register-166398357` branch
* Clone the repo
* Run `npm install`
* Run `npm test`
* All the tests should fail
#### What are the relevant pivotal tracker stories?
* [#166398357](https://www.pivotaltracker.com/story/show/166398357)
